metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -541,7 +541,9 @@ class MetalsLanguageServer(
       capabilities.setDocumentOnTypeFormattingProvider(
         new DocumentOnTypeFormattingOptions("\n")
       )
-      capabilities.setDocumentRangeFormattingProvider(true)
+      capabilities.setDocumentRangeFormattingProvider(
```

Comment:
So probably the best that we can do currently is add an option to MetalsServerConfig like:
```scala
  allowMultilineStringFormatting = MetalsServerConfig.binaryOption(
      "metals.allow-multiline-string-formatting",
      default = true
    ),
```
And then:
```scala      

capabilities.setDocumentRangeFormattingProvider(initialConfig.allowMultilineStringFormatting)
```